src/main/client/player/player-manager.ts

```diff
@@ -2,30 +2,27 @@ import { Inject, Singleton } from 'typescript-ioc';
 import { PlayerComponent } from './player-component';
 import { GameStateComponent } from '../game-state/game-state-component';
 import { GameComponent } from '../game/game-component';
+import { NetworkComponent } from '../network/network-component';
 
 @Singleton
 export class PlayerManager {
    constructor(
       @Inject private readonly component: PlayerComponent,
       @Inject private readonly gameState: GameStateComponent,
       @Inject private readonly game: GameComponent,
+      @Inject private readonly network: NetworkComponent,
    ) {
-      gameState.gameStarted$.subscribe(() => {
-         this.mockClient();
+      network.loginResponse$.subscribe((response) => {
+         console.log('My id is ', response.id);
```

Comment:
Be sure to remove this later, maybe better logging should be needed

Reply:
Sure. Console logs can be removed in prod by webpack, though.